ui/src/app/pages/homepage/quick-tour-modal.spec.tsx

```diff
@@ -32,49 +29,53 @@ describe('QuickTourModalComponent', () => {
 
   it('should not show the previous button when we are on the first slide', () => {
     component();
-
     expect(
       screen.queryByRole('button', { name: /previous/i })
     ).not.toBeInTheDocument();
   });
 
   it('should go to the next slide when we click next', () => {
-    const { getByTestId } = component();
-    expect(getByTestId('panel-title').textContent).toBe(panels[0].title);
+    component();
+    expect(screen.getByTestId('panel-title').textContent).toBe(panels[0].title);
     fireEvent.click(screen.getByRole('button', { name: /next/i }));
-    expect(getByTestId('panel-title').textContent).toBe(panels[1].title);
+    expect(screen.getByTestId('panel-title').textContent).toBe(panels[1].title);
   });
 
   it('should go to the panel we select from the breadcrumbs', () => {
-    const { getByTestId } = component();
+    component();
     const panelNum = 2;
-    fireEvent.click(getByTestId(`breadcrumb${panelNum}`));
-    expect(getByTestId('panel-title').textContent).toBe(panels[panelNum].title);
+    fireEvent.click(screen.getByTestId(`breadcrumb${panelNum}`));
+    expect(screen.getByTestId('panel-title').textContent).toBe(
+      panels[panelNum].title
+    );
   });
 
   it('should go to the previous slide when we click previous', () => {
-    const { getByTestId } = component();
-    expect(getByTestId('panel-title').textContent).toBe(panels[0].title);
+    component();
+    expect(screen.getByTestId('panel-title').textContent).toBe(panels[0].title);
     fireEvent.click(screen.getByRole('button', { name: /next/i }));
-    expect(getByTestId('panel-title').textContent).toBe(panels[1].title);
+    expect(screen.getByTestId('panel-title').textContent).toBe(panels[1].title);
     fireEvent.click(screen.getByRole('button', { name: /previous/i }));
-    expect(getByTestId('panel-title').textContent).toBe(panels[0].title);
+    expect(screen.getByTestId('panel-title').textContent).toBe(panels[0].title);
   });
 
   it('should not show the next button when we are on the last slide', () => {
-    const { getByTestId } = component();
-    fireEvent.click(getByTestId(`breadcrumb${lastPanel}`));
-    expect(screen.getByRole('button', { name: /close/i })).toBeInTheDocument();
+    component();
+    const lastPanel = panels.length - 1;
+    fireEvent.click(screen.getByTestId(`breadcrumb${lastPanel}`));
+    expect(
```

Comment:
I ran a few times, and it all passed..do u have a better way to test this out?

Reply:
I think that this would give it a chance to disappear if that does not happen instantaneously
`await waitForElementToBeRemoved(() => screen.queryByRole('button', { name: /next/i }))`

You could add this back to the end if you want to make sure that the test has an expect statement:
`expect(screen.getByRole('button', { name: /close/i })).toBeInTheDocument();`

Reply:
does [this](https://github.com/all-of-us/workbench/pull/8507/commits/02085a8512c9814e6b4ba15d293b12e86b67fea6) work? basically saying we should see `close`, but we don't see `next`

Reply:
ah ok...I thought `getByRole` waits....

Reply:
No worries. Only find waits:
https://testing-library.com/docs/react-testing-library/cheatsheet/#queries